### PR TITLE
fix: wrong path to service property

### PR DESCRIPTION
**Pull request by craicoverflow**
The path to the Datasync service was wrong (my bad).

this is not easily verifiable unless we add a data sync template.

### CODE REVIEW

**Comment by machi1990** [comment]
> The path to the Datasync service was wrong (my bad).
> 
> this is not easily verifiable unless we add a data sync template.

Yeap. Or have some sort of integration test for it.

**Comment by ssd71** [comment]
+1 for template, I believe integration test is also needed, though separate issues

File: packages/graphback-runtime/src/resolvers/LayeredRuntimeResolverCreator.ts [hunk 1]
```diff
@@ -89,7 +89,7 @@ export class LayeredRuntimeResolverCreator {
         const deltaQuery = getDeltaQuery(resolverElement.graphqlType.name)
 
         resolvers.Query[deltaQuery] = async (parent: any, args: any, context: any) => {
```
Comment (machi1990) [inline packages/graphback-runtime/src/resolvers/LayeredRuntimeResolverCreator.ts:91]: I am thinking, would having a "full type context object" helped here?.

Comment (craicoverflow) [reply packages/graphback-runtime/src/resolvers/LayeredRuntimeResolverCreator.ts:91]: probably yep, I will add it :)

**Pull request merged**